modules/social_features/social_landing_page/templates/group--featured--sky.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends 'group--featured.html.twig' %}
```

Comment:
I feel like this file has a lot in common with changes we make in #1511 Perhaps we can deduplicate this somehow?

Additionally, we shouldn't have `--sky` things in modules since this is a theme thing.

Reply:
A template was refactored and simplified.